app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def hello
+    render html: "hello, world!"
```

Comment:
use ' instead of " , read guideline here - https://github.com/bbatsov/ruby-style-guide